pkg/download/automation/internal/jinja.go

```diff
@@ -0,0 +1,28 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package internal
+
+import "bytes"
+
+// EscapeJinjaTemplates replaces each occurrence of "{{" with "\{\{" and each occurrence of "}}" with "\}\}"
```

Comment:
Comment is outdated. Also it would be great if it explains why the "pre-escaping" is needed :) 

Also is my assumption correct, that we don't actually need the `{{` -> `\{\{` escaping and just do it for completeness?
We'll only run into issues with the closing bracket escaping, as it would wrongly also escape the closing bracket of the previously added `{{'{{'}}`, right?

Reply:
done